exporter/prometheusremotewriteexporter/config.go

```diff
@@ -42,6 +47,19 @@ type Config struct {
 	exporterhelper.ResourceToTelemetrySettings `mapstructure:"resource_to_telemetry_conversion"`
 }
 
+// QueueConfig allows to configure the remote write queue.
+type QueueConfig struct {
+	// Size is the maximum number of OTLP metric batches allowed
+	// in the queue at a given time.
+	Size int `mapstructure:"size"`
+
+	// Concurrency configures the number of workers used by
+	// the collector to fan out remote write requests.
+	Concurrency int `mapstructure:"concurrency"`
```

Comment:
If this is not same as PRW, maybe use NumWorkers to be consistent with QueueRetry config. If it is same as PRW just let me know and we can ignore the comment.

Reply:
You mean  NumConsumer, right? I should probably rename the Size to QueueSize to keep things consistent.

Reply:
PTAL.

Renamed the queue_config to remote_write_queue. Renamed concurrency to num_consumers and size to queue_size. This looks much better and is more honest.